src/helper/client.cpp

```diff
@@ -466,6 +469,16 @@ bool client::handle_command(network::request_shutdown::request &command)
 
             response->triggers = std::move(res->events);
             response->force_keep = res->force_keep;
+            for (const auto &[key, value] : res->schemas) {
+                std::string schema = value;
+                if (value.length() > max_plain_schema_allowed) {
+                    auto encoded = compress(schema);
```

Comment:
Compress should also accept a `std::string_view` to avoid copying the schema into an `std::string`